test/Renderer/ImageTest.php

```diff
@@ -38,7 +38,8 @@ public function testType()
     public function testGoodImageResource()
     {
         $imageResource = imagecreatetruecolor(1, 1);
-        $this->renderer->setResource($imageResource);
+        $renderer = $this->renderer->setResource($imageResource);
+        $this->assertSame($this->renderer, $renderer);
```

Comment:
These lines are not part of the solution. Right?
It would great if you can create a separate commit or pull request for this type of changes. This helps on reviewing, improves the commit history and allows cherry-picking.

Thanks in advance!

Reply:
Removed.